### ISSUE TITLE
Is it possible to have tabs or plugins in an ice firefox profile?

### ISSUE THREAD

**Issue by vonpupp** [peppermintos/ice#7]
Hello,

I like the idea of having privacy invasive webapps (like, google, facebook, among others) within ice, however I find the lack of tab support not to be very convenient.

Is there a way to have tab support? (be able to use regular tabs). Would it be possible to use plugins? If it would be the case, vimperator would do the trick for me.

Thank you.

**Comment by Mark-Peppermint** [comment]
The lack of tabs is kinda the point, but that doesn't mean you cannot navigate hidden tabs .. see if this helps:
https://forum.peppermintos.com/index.php/topic,4314.msg42956.html#msg42956

Plugins - YES Firefox SSB's can have plugins on a "per SSB" basis (different plugins for different SSB's .. installing them will hopefully become clear when you've checked the above link.

**Comment by vonpupp** [comment]
Thank you Mark,

I imagine that my use case is not the common use case, however I think this could benefit others also and attract more audience.

I saw the link and it seems to be a good solution however when I try to use the key shortcuts (i.e: `ctrl` + `tab`), as soon as I get to the last tab, I cannot move any further (i.e: go to the first tab). `Alt` + `left/right` do not seem to work on my SSB, it works fine on regular firefox however.

I could install vimperator, however I have another issue but it is not related to ice. My configs for vimperator are quite paranoid as default: not to save anything (cookies, history, etc). This is fine for regular browsing but with SSB I would like to save regular history and cookies. I will give it a second thought to see if I can use SSB only with another browser instead of my main browser (firefox). The last time I tried (I use Arch Linux), I only had the `ice-firefox` command but nothing else.

I have command wrappers around the sites I want to use on my bin folder, for instance:

```
> $ cat bin/ice-facebook.sh                                                    
ice-firefox facebook.com &
```

I will think again how to try to solve this out. Thanks a lot!
